include/fiction/algorithms/physical_design/post_layout_optimization.hpp

```diff
@@ -1072,8 +784,8 @@ bool compare_gates(const tile<Lyt>& a, const tile<Lyt>& b)
  * creates more compact layouts by freeing up space to the right and bottom, as all gates were moved to the top left
  * corner.
  *
- * After moving all gates, this algorithm also checks if excess wiring exists on the layout, i.e., rows that only
- * contain vertical wires or columns that only contain horizontal wires and removes them.
+ * After moving all gates, this algorithm also checks if excess wiring exists on the layout using the wiring_reduction
```

Comment:
`wiring_reduction` or maybe you wanna add something like "(cf. `wiring_reduction.hpp`)"